spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::SpecificMatcher do
+  it 'does not register an offense for abstract matcher when elements with ' \
+    'no matcher specified in detail' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_selector('article')
+      expect(page).to have_no_selector('body')
+      expect(page).to have_css('tbody')
+    RUBY
+  end
+
+  it 'does not register an offense for abstract matcher when elements with ' \
+    'not element' do
```

Comment:
I’m not sure what “when elements with not element” means.

Reply:
Improved wording. What do you think?

https://github.com/rubocop/rubocop-rspec/blob/6fb113cdc3dc298f95359a5519fd6050280ce98f/spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb#L4-L5

https://github.com/rubocop/rubocop-rspec/blob/6fb113cdc3dc298f95359a5519fd6050280ce98f/spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb#L13-L14